web/client/components/map/openlayers/DrawSupport.jsx

```diff
@@ -253,7 +253,7 @@ export default class DrawSupport extends React.Component {
                         });
                     } else {
                         feature = new Feature({
-                            geometry: this.createOLGeometry(geometry.geometry ? geometry.geometry : {...geometry, ...geometry.properties, center })
+                            geometry: this.createOLGeometry(geometry.geometry ? geometry.geometry : {...geometry, ...omit(geometry.properties, "type"), center })
                         });
```

Comment:
I noticed that 
```javascript
export const createOLGeometry = ({ type, coordinates, radius, center } = {}) => {
```
So we should get at least only radius or center, (or omit also coordinates) to avoid other possible name mismatch.